src/attachment/attachment-transformer.ts

```diff
@@ -0,0 +1,68 @@
+import { MarkdownAbstractSyntaxTree, RenderProps } from "react-markdown";
+const visit = require("unist-util-visit"); //TODO define proper types for this or add types dev dependency
```

Comment:
Is there a reason `require` is used here instead of an `import`?

Reply:
Yeah I forgot to update that :). I always have problems finding/getting types for things.